ui/ducks/swaps/swaps.js

```diff
@@ -584,7 +585,7 @@ export const fetchSwapsLivenessAndFeatureFlags = () => {
         await dispatch(fetchSmartTransactionsLiveness());
         const transactions = await getTransactions({
           searchCriteria: {
-            from: state.metamask?.selectedAddress,
+            from: getSelectedInternalAccount(state).address,
```

Comment:
Ultimately this should never be `undefined` right? The previous implementation could be (`state.metamask?.selectedAddress`), should we keep this behavior here too?

Reply:
Yea, it shouldn't, but it makes sense to keep this behavior